### ISSUE TITLE
Splinter crashes when site uses bugsnag

### ISSUE THREAD

**Issue by minthemiddle** [cobrateam/splinter#376]
When a site uses the error monitoring tool [Bugsnag](https://bugsnag.com/), all 'Splinter' methods for finding elements crash. I'm using the latest Splinter via `pip install splinter`.

Here is very basic example with a site that I found that uses Bugsnag:

```
from splinter import Browser
browser = Browser()
browser.visit('http://www.urlaubspiraten.de')
headings = browser.find_by_css('.post-hl-lnk')
```

This will return an error that ends with:

```
WebDriverException: Message: Permission denied to access property 'handleEvent'
Stacktrace:
    at <anonymous> (http://d2wy8f7a9ursnm.cloudfront.net/bugsnag-2.min.js:1)
```

When I save the page and remove the file `bugsnag-2.min.js` manually, I get a list of results.

**Comment by irusev** [comment]
I have the same issue using Selenium WebDriver - getting absolutely the same error when testing a site using Bugsnag. Did you find any workaround or something useful?

ivan

**Comment by martinbetz** [comment]
No, I did not find a workaround. Obviously it is a bug in Selenium as I had the same issue when using vanilla Selenium. One of us should file a bug report at their repo.

**Comment by irusev** [comment]
I'm not sure that I can provide them with exact example to reproduce the problem. Something interesting that I found is that you get the error only the first time after page load. If you catch the error and re-try searching for the element, then it's returned just fine with no error.

If you can file a bug against Selenium, that would be great!

Best Regards,
Ivan

**Comment by ConradIrwin** [comment]
This is fixed in Bugsnag v2.4.8
